### DISCUSSION TITLE
peek the help of selected directive

### DISCUSSION THREAD

**Discussion by hxy1027** [marp-team/marp#214] [Q&A] [answered]
Hit Ctrl + Space within the front-matter or HTML comment to show the list of directives.  You can peek the help of selected directive by hitting Ctrl + Space again.
when i hit Ctrl + Space ,there is no response, do i need to set something in vscode,thanks

**Comment by yhatt** [comment, marked as answer]
You may have been conflicted VS Code key binding with OS key bindings. e.g. Toggle IME status for CJK language, Open Spotlight search in macOS, etc.

You can find out or set an alternate key combination in VS Code ([Refer to VS Code documentation](https://code.visualstudio.com/docs/getstarted/keybindings)), or disable the shortcut in setting of the computer, to pass the key combination into VS Code rightly.

- [visual studio code - How do I trigger suggestion(s) in VSCode in Catalina? - Stack Overflow](https://stackoverflow.com/questions/59505662/how-do-i-trigger-suggestions-in-vscode-in-catalina/59505958#59505958)
- [visual studio code - VisualCode Shortcut to display autocomplete list before typing in MacOS - Stack Overflow](https://stackoverflow.com/questions/51341555/visualcode-shortcut-to-display-autocomplete-list-before-typing-in-macos/59774317)
- [javascript - How to trigger vscode intellisense using keyboard on osx? - Stack Overflow](https://stackoverflow.com/questions/56143239/how-to-trigger-vscode-intellisense-using-keyboard-on-osx/59774240#59774240)
- https://github.com/microsoft/vscode/issues/73557
- https://github.com/marp-team/marp-vscode/issues/242
